src/redis_request.cc

```diff
@@ -35,7 +35,7 @@
 namespace Redis {
 const size_t PROTO_INLINE_MAX_SIZE = 16 * 1024L;
 const size_t PROTO_BULK_MAX_SIZE = 512 * 1024L * 1024L;
-const size_t PROTO_MULTI_MAX_SIZE = 1024 * 1024L;
+const int64_t PROTO_MULTI_MAX_SIZE = 1024 * 1024L;
```

Comment:
Why do you make this change?

Reply:
compiler warning. `size_t == int64_t`

Reply:
You may instead change `multi_bulk_len_ > PROTO_MULTI_MAX_SIZE` to `multi_bulk_len_ > (int64_t) PROTO_MULTI_MAX_SIZE`?

Reply:
I also think it is better to keep `PROTO_MULTI_MAX_SIZE` typed `size_t`.

Reply:
yes, you are right, it should be positive constant, thank you.